fern/products/sdks/overview/csharp/version-compatibility.mdx

```diff
@@ -0,0 +1,270 @@
+---
+title: .NET Framework compatibility
```

Comment:
Let's shorten it to `.NET compatibility`, it looks a little awkward
<img width="360" height="98" alt="Screenshot 2025-10-24 at 1 45 22 PM" src="https://github.com/user-attachments/assets/6bfc937a-20c2-4125-81cf-d441e8de3055" />